gradle.properties

```diff
@@ -1,5 +1,5 @@
 spongeApiVersion = 7.3.0
-nucleusVersion = 2.3.3
+nucleusVersion = 2.3.4
```

Comment:
Do not change version numbers - this is a snapshot for 2.3.3 which hasn't been released yet. These should only be changed during release cycles.